src/with_clause_parser.c

```diff
@@ -113,6 +114,24 @@ ts_with_clauses_parse(const List *def_elems, const WithClauseDefinition *args, S
 	return results;
 }
 
+extern TSDLLEXPORT char *
+ts_with_clause_result_unparse_value(const WithClauseResult *result)
+{
+	Oid oid = result->definition->type_id;
+	if (!OidIsValid(oid))
+		elog(ERROR, "Argument \"%d\" has invalid Oid!", oid);
```

Comment:
[Error Message Style Guide](https://www.postgresql.org/docs/current/error-style-guide.html)

```suggestion
		elog(ERROR, "argument \"%d\" has invalid OID", oid);
```